framework/build.xml

```diff
@@ -362,6 +364,45 @@
         </antcall>
     </target>
 
+    <target name="signature-inference-tests" depends="jar,build-tests"
+            description="Run tests for signature inference using .jaif files">
+        <delete dir="build/signature-inference"/>
+        <!--Runs the checker on test files to find expected errors and generate
```

Comment:
This comment in checker-framework/framework/build.xml suggests that running SignatureInferenceTest finds expected errors, but I don't see how that happens.  Were they manually written in file ExpectedErrors.java?